### ISSUE TITLE
Feature request: Include gdb (with python support)

### ISSUE THREAD

**Issue by Grazfather** [nongiach/arm_now#24]
```(gdb) pi
>Scripting in the "Python" language is not supported in this copy of GDB.
```

Would be great if gdb were compiled with python. Now sure how involved this would be.

**Comment by nongiach** [comment]
Hey,
What I recommend is to use gdb-server and then using gdb-multiarch to connect to the target.
This way you have all the python you want without getting a lot of complication inside the VM.
Check out the port forwarding option:
https://github.com/nongiach/arm_now/wiki/2.2-Network:-Internet-and-port-forwarding

**Comment by Grazfather** [comment]
Yeah fair enough, but then my request would be to include gdbserver :)

**Comment by nongiach** [comment]
Yep, that's right. It's in coming.
